### PR TITLE
Updates classes for Blog post cards in styles and templates

### LINKED ISSUE #87: Japanese ubuntu.com blog - update templates
implement [design](https://github.com/canonical-websites/jp.ubuntu.com/issues/86)

### PR DESCRIPTION
**Pull request by b-m-f**
## Done
Uses clearly identifiable classes for Blog content

## QA

Activate the blueprint for the blog in `app.py` with 
```
from canonicalwebteam.blog.app import BlogExtension
blog = BlogExtension()
blog.init_app(app, "Blog title", [2996], "jp.ubuntu.com", "/blog")
```

## Screenshots

image

Fixes #87

### CODE REVIEW

**Comment by barrymcgee** [comment]
@b-m-f Your scss-lint check is failing.

File: static/sass/_patterns_blog-card.scss [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+      border-top: 1px dotted $color-mid-light;
+    }
+  }
+
+  .blog-p-card__header {
+    @extend %post-card-header;
+
+    &--cloud-and-server {
+      @extend %post-card-header;
+      border-top: 3px solid #a87ca0;
```
Comment (barrymcgee) [inline static/sass/_patterns_blog-card.scss]: Why are we using custom hex values here?

**Comment by barrymcgee** [comment]
@b-m-f Approved with just one query.

File: static/sass/_patterns_blog-card.scss (hunk 1 above)
Comment (b-m-f) [reply static/sass/_patterns_blog-card.scss]: I took the code from https://github.com/canonical-websites/blog.ubuntu.com/blob/master/static/sass/_pattern_card.scss . 
Did not update them. 
I could create an Issue for this to fix it in another PR.

**Pull request merged**